src/components/ServiceRequestForm.jsx

```diff
@@ -221,12 +221,14 @@ const ServiceRequestForm = (props, errors, touched) => {
 		SetFieldValues(localProps, requestSubFields);
 
 		if (subInfo !== undefined) {
-			if (subInfo.parentId !== undefined) {
-				SetFieldValues(localProps, { requestTypeParentID: subInfo.parentId });
-			}
-			if (subInfo.parentName !== undefined) {
-				SetFieldValues(localProps, { requestTypeParent: subInfo.parentName });
-			}
+			(subInfo.parentId !== undefined) ? 
```

Comment:
I think whenever i suggested to use ternaries I think i gave you the idea.

In this case you could do

```js
const parentRequestTypeId = subInfo.parentId !== undefined 
  ? subInfo.parentId 
  : "";
SetFieldValues(localProps, { requestTypeParentID: parentRequestTypeId });
```

The benefit of this is

- If more logic is added to the `parentRequestTypeId` we could refactor it out, but wouldn't have to change teh SetFieldValues line
- We only need to call SetFieldValues once
- We only declare the object once `{ requestTypeParentId}`

Reply:
I cannot find this specific advice in the airbnb [styleguide](https://github.com/airbnb/javascript), so it might be good to document this somewhere.

Reply:
done